examples/kfto-dreambooth/README.md

```
@@ -13,6 +13,18 @@ This example is based on HuggingFace DreamBooth Hackathon example - https://hugg
 * Sufficient worker nodes for your configuration(s) with NVIDIA GPUs (Ampere-based or newer recommended) or AMD GPUs (AMD Instinct MI300X or newer recommended)
 * AWS S3 storage available
 
+> [!IMPORTANT]
+> **Kueue Integration (RHOAI 2.21+):**
+> * If using RHOAI 2.21+, the example supports Kueue integration for workload management:
+>   * When using Kueue:
+>     * Follow the [Configure Kueue (Optional)](#configure-kueue-optional) section to set up required resources
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)